contracts/axelarnet-gateway/src/contract/execute.rs

```diff
@@ -79,6 +80,7 @@ impl CallContractData {
 enum RoutingDestination {
     Nexus,
     Router,
+    Axelarnet,
```

Comment:
```suggestion
    This,
```
might be weird lol but I prefer not having to think what `Axelarnet` v.s. `Nexus` is.